dist/style.css

```diff
@@ -292,5 +295,8 @@ a {
 }
 
 hr {
-  width: 50%;
+  display:block;
```

Comment:
Put a space between `display` and `block;`

I need to get you guys a linter to point out syntax errors. Coming soon.